apiary_v2.apib

```diff
@@ -1231,4 +1231,670 @@ the listed orders by their creation date.
           }
 
 
+# Group Factory API
 
+After an Ecommerce order has been created, Unmade begins a process of converting the various line items in the order 
+into production jobs which can then be manufactured at your vendor factory. The Factory API is used to interact with
+all order data related to this production process.
+
+Each Ecommerce Order maps to exactly one Factory API Order. The same order ID can be used in the order endpoints on both
+APIs.
+
+The Factory API is accessed via `https://factory.unmade.com/`. All requests require your authorization token which is 
+passed as an HTTP request header.
+
+## Order States
+
+During production, orders progress through the following states:
+
+ - `new`: The order has not yet begun production
+ - `in_production`: The order has been accepted and is now in production
+ - `ready-to-ship`: The order has been produced and is awaiting shipping
+ - `shipped`: The order has left the factory
+ - `cancelled`: The order has been cancelled
```

Comment:
I like that here we use `-` instead of `_` like on the ecommerce side

Reply:
Which do we want to use? Underscores everywhere?

Reply:
That will be to personal preference TBH ... it feels nicer `-` but I'm happy with just one and the same across both systems 😄

Reply:
Everything is underscore, apart from Factory Order `ready-to-ship`. I think we should go all underscore and bulk-update Factory orders when we get there.